pkg/binutils/processes.go

```diff
@@ -111,12 +114,18 @@ func GetServerPID() (int, error) {
 
 // StartServerProcess starts the gRPC server as a reentrant process of this binary
 // it just executes `avalanche-cli backend start`
-func StartServerProcess(log logging.Logger) error {
+func StartServerProcess(app app.Avalanche) error {
 	thisBin := reexec.Self()
 
 	args := []string{"backend", "start"}
 	cmd := exec.Command(thisBin, args...)
-	outputFile, err := os.CreateTemp("", "avalanche-cli-backend*")
+
+	outputFilePrefix := path.Join(app.GetRunDir(), "server")
```

Comment:
why a dir with tmp for each execution? maybe only a dir like server-info, and inside it the different execution files with timestamps?